xml/wgl.xml

```diff
@@ -72,44 +72,44 @@ Registry at https://github.com/KhronosGroup/OpenGL-Registry
-        <enum value="0x00800000"  name="WGL_SWAP_UNDERLAY8"/>
-        <enum value="0x01000000"  name="WGL_SWAP_UNDERLAY9"/>
-        <enum value="0x02000000"  name="WGL_SWAP_UNDERLAY10"/>
-        <enum value="0x04000000"  name="WGL_SWAP_UNDERLAY11"/>
-        <enum value="0x08000000"  name="WGL_SWAP_UNDERLAY12"/>
-        <enum value="0x10000000"  name="WGL_SWAP_UNDERLAY13"/>
-        <enum value="0x20000000"  name="WGL_SWAP_UNDERLAY14"/>
-        <enum value="0x40000000"  name="WGL_SWAP_UNDERLAY15"/>
+        <enum value="0x00000001"  name="WGL_SWAP_MAIN_PLANE" group="Plane"/>
+        <enum value="0x00000002"  name="WGL_SWAP_OVERLAY1" group="Plane"/>
+        <enum value="0x00000004"  name="WGL_SWAP_OVERLAY2" group="Plane"/>
+        <enum value="0x00000008"  name="WGL_SWAP_OVERLAY3" group="Plane"/>
+        <enum value="0x00000010"  name="WGL_SWAP_OVERLAY4" group="Plane"/>
+        <enum value="0x00000020"  name="WGL_SWAP_OVERLAY5" group="Plane"/>
+        <enum value="0x00000040"  name="WGL_SWAP_OVERLAY6" group="Plane"/>
+        <enum value="0x00000080"  name="WGL_SWAP_OVERLAY7" group="Plane"/>
+        <enum value="0x00000100"  name="WGL_SWAP_OVERLAY8" group="Plane"/>
+        <enum value="0x00000200"  name="WGL_SWAP_OVERLAY9" group="Plane"/>
+        <enum value="0x00000400"  name="WGL_SWAP_OVERLAY10" group="Plane"/>
+        <enum value="0x00000800"  name="WGL_SWAP_OVERLAY11" group="Plane"/>
+        <enum value="0x00001000"  name="WGL_SWAP_OVERLAY12" group="Plane"/>
+        <enum value="0x00002000"  name="WGL_SWAP_OVERLAY13" group="Plane"/>
+        <enum value="0x00004000"  name="WGL_SWAP_OVERLAY14" group="Plane"/>
+        <enum value="0x00008000"  name="WGL_SWAP_OVERLAY15" group="Plane"/>
+        <enum value="0x00010000"  name="WGL_SWAP_UNDERLAY1" group="Plane"/>
+        <enum value="0x00020000"  name="WGL_SWAP_UNDERLAY2" group="Plane"/>
+        <enum value="0x00040000"  name="WGL_SWAP_UNDERLAY3" group="Plane"/>
+        <enum value="0x00080000"  name="WGL_SWAP_UNDERLAY4" group="Plane"/>
+        <enum value="0x00100000"  name="WGL_SWAP_UNDERLAY5" group="Plane"/>
+        <enum value="0x00200000"  name="WGL_SWAP_UNDERLAY6" group="Plane"/>
+        <enum value="0x00400000"  name="WGL_SWAP_UNDERLAY7" group="Plane"/>
+        <enum value="0x00800000"  name="WGL_SWAP_UNDERLAY8" group="Plane"/>
+        <enum value="0x01000000"  name="WGL_SWAP_UNDERLAY9" group="Plane"/>
+        <enum value="0x02000000"  name="WGL_SWAP_UNDERLAY10" group="Plane"/>
+        <enum value="0x04000000"  name="WGL_SWAP_UNDERLAY11" group="Plane"/>
+        <enum value="0x08000000"  name="WGL_SWAP_UNDERLAY12" group="Plane"/>
+        <enum value="0x10000000"  name="WGL_SWAP_UNDERLAY13" group="Plane"/>
+        <enum value="0x20000000"  name="WGL_SWAP_UNDERLAY14" group="Plane"/>
+        <enum value="0x40000000"  name="WGL_SWAP_UNDERLAY15" group="Plane"/>
     </enums>
 
     <enums namespace="WGLColorBufferMask" type="bitmask" vendor="ARB">
```

Comment:
It looks like a pretty decent attempt has been made when originally writing the XML spec to group up each enum into namespaces. Can we have the group attributes make use of these existing namespaces? (it's better if we reuse as much ahead-of-time bikeshedding of the names as possible, given these names are already approved)

Reply:
Look at that, totally missed these when doing the PR... 😅